lib/tessel/update.js

```diff
@@ -77,20 +79,21 @@ Tessel.prototype.updateOpenWRT = function(opts, image) {
     .then(() => {
       return new Promise((resolve, reject) => {
         // Begin the sysupgrade
-        logs.info('Starting OpenWRT update.');
-        logs.info('Please do not remove power from Tessel.');
-        logs.info('This process will take at least two minutes...');
+        log.info('Starting OpenWRT update.');
+        log.info('Please do not remove power from Tessel.');
+        log.info('This process will take at least 2 minutes...');
+        log.spinner.start();
 
         var sysupgradeCommand = [];
         if (opts.n) {
-          logs.info('Configuration is not saved during update.');
+          log.info('Configuration is not saved during update.');
           sysupgradeCommand = commands.sysupgradeNoSaveConfig(updatePath);
         } else {
-          logs.info('Configuration is saved during update.');
+          log.info('Configuration is saved during update.');
           sysupgradeCommand = commands.sysupgrade(updatePath);
         }
 
-        // The USBDaemon will cut out or the SSH command will close        
+        // The USBDaemon will cut out or the SSH command will close
         this.connection.exec(sysupgradeCommand, (err, remoteProc) => {
           if (err) {
             return reject(err);
```

Comment:
I've updated this so that both `closeSuccessfulCommand` and `closeFailedCommand` can stop the spinner